src/components/common/Certify.vue

```diff
@@ -3,20 +3,80 @@
     <header>
       <h2>2. Certify</h2>
     </header>
-
     <span>Enter the legal name of the person authorized to complete and submit these changes.</span>
-
-    <div id="certify-container" class="mt-4">
-      Under construction!
-    </div>
+    <certify
+      :currentDate="readableCurrentDate"
+      :certified-by="getCertifyState.certifiedBy"
+      :is-certified="getCertifyState.valid"
+      :entity-display="readableEntityType"
+      :message="certifyMessage"
+      @update:certifiedBy="onCertifiedBy($event)"
+      @update:isCertified="onValid($event)"
+    />
   </section>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Mixins } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+
+// Components
+import { Certify } from '@bcrs-shared-components/certify'
+
+// Mixins
+import { DateMixin, EnumMixin } from '@/mixins'
+
+// Interfaces, Enums & Resources
+import { ActionBindingIF, CertifyIF, CertifyStatementIF } from '@/interfaces'
+import { EntityTypes } from '@/enums'
+
+@Component({
+  components: {
+    Certify
+  }
+})
+export default class CertifySection extends Mixins(DateMixin, EnumMixin) {
```

Comment:
Please rename the file too :)

Reply:
Recommendations?  We can't have 2 `Certify` components :P

Reply:
Actually we can, because this file is the section and the shared component isn't in this repo.

But since you renamed this class, could you also rename this file to `CertifySection.vue` ?

Reply:
I suppose we could but when importing them both for sake of testing, how can we differentiate one from the other there?

Reply:
Let's chat in person.

Reply:
Copy that!